src/micro.c

```diff
@@ -23,12 +23,14 @@ static inline void _unlock_service(microService *m) { natsMutex_Unlock(m->servic
 
 static microError *_clone_service_config(microServiceConfig **out, microServiceConfig *cfg);
 static microError *_new_service(microService **ptr, natsConnection *nc);
-static microError *_wrap_connection_event_callbacks(microService *m);
+static void _on_connection_closed(natsConnection *nc, void *ignored);
```

Comment:
nit: here you use `ignored` and the next you use `not_used`, could make them the same :-)